### PR TITLE
english version 初稿完成

### PR DESCRIPTION
**Pull request by qq6690876**
english version 初稿完成

### CODE REVIEW

File: docs/guide/sheet.md [hunk 1]
```diff
@@ -512,30 +512,30 @@ eg: options.data：
...
-            "foot": { //页脚颜色
+            "foot": { //Footers color
                 "fc": "#000",
                 "bc": "#a5efcc"
             }
         },
-        "hasRowHeader": false, //含有页眉
-        "hasRowFooter": false //含有页脚
+        "hasRowHeader": false, //is included header
+        "hasRowFooter": false //is included footers
```
Comment (Dushusir) [inline docs/guide/sheet.md:538]: footers changed to footer

File: docs/guide/sheet.md [hunk 2]
```diff
@@ -1248,18 +1247,18 @@ Luckysheet在初始化完成之后进行的一系列操作，会将更多本地
...
-        "visibledatacolumn": [], //所有列的位置
-        "ch_width": 2322, //工作表区域的宽度
-        "rh_height": 949, //工作表区域的高度
-        "load": "1", //已加载过此sheet的标识
-        "data": [], //更新和存储使用的单元格数据
+        "visibledatarow": [], //positions of all rows
+        "visibledatacolumn": [], //positions of all columns
+        "ch_width": 2322, //The width of a sheet
+        "rh_height": 949, //The height of a sheet
+        "load": "1", //check whether this sheed loaded
```
Comment (Dushusir) [inline docs/guide/sheet.md:1260]: How about this： `Check whether this sheet has been loaded`

**Review by Dushusir — CHANGES_REQUESTED** [review]

File: docs/guide/sheet.md [hunk 3]
```diff
@@ -114,13 +114,13 @@ eg: options.data：
 ### celldata
 - type：Array
 - default：[]
-- usage： 原始单元格数据集，存储sheet中所有单元格中的值，是一个包含`{r:0,c:0,v:{m:"value",v:"value",ct: {fa: "General", t: "g"}}}`格式单元格信息的一维数组，只在初始化的时候使用。
+- usage： The original cell data set is a set containing `{r:0,c:0,v:{m:"value",v:"value",ct: {fa: "General", t: "g"}} }`The one-dimensional array of format cell information is only used during initialization.
 
-    r代表行，c代表列，v代表该单元格的值，值可以是字符、数字或者对象。
+    `r` represents the row, `c` represents the column, and `v` represents the value of the cell. value could be char, number, or object
```
[the remark this answers is not in the document]
Comment (qq6690876) [reply docs/guide/sheet.md:119]: thanks

**Comment by qq6690876** [comment]
create a new request

**Pull request closed without merging**